config/dev.json

```diff
@@ -2,6 +2,7 @@
   "agoraUrlRoot": "https://agora.dsde-dev.broadinstitute.org",
   "dockstoreUrlRoot": "https://staging.dockstore.org:8443",
   "googleClientId": "500025638838-s2v23ar3spugtd5t2v1vgfa2sp7ppg0d.apps.googleusercontent.com",
+  "googleCloudBillingKey": "AIzaSyDdGqv2Jcsv9CUd9Km0XhdavdPtrOAlZWU",
```

Comment:
Is this ok to have in source control?

Reply:
I can't think of a way to keep this a secret, so I wouldn't want to treat it as a secret.

Reply:
(Because it has to be in the call URL)

Reply:
The alternative was to pull the relevant part of this JSON during the deploy, and have a dummy json for dev.